storybook/preview.js

```diff
@@ -114,8 +114,7 @@ export const parameters = {
 				<Subtitle />
 				<Primary />
 				<Description />
-				{ /* `story="^"` enables Controls for the primary props table */ }
-				<ArgsTable story="^" />
+				<Controls />
```

Comment:
`ArgsTable` was removed in favor of `Controls` and `ArgTypes`, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#argstable-doc-block-removed